_sass/landing-page.scss

```diff
@@ -254,20 +254,36 @@
     color: #fff;
   }
 
+  .mail-row p a{
+    margin: auto;
+    display: block;
+    width: auto;
+  }
```

Comment:
I don't think these set of CSS are required, please confirm and remove them

Reply:
Hey @Nikhil-Ladha, If I remove this part then the text is not centered. Particularly if I remove `display: block` then the text is no longer remaining center aligned.

Reply:
I have experimented with this part. So, I guess we have to keep iy.

Reply:
Sometimes doing a sanity check helps a lot :)
Did you notice that the CSS from lines: 263 to 272 is not being applied to the code at all? I don't think you have!
And, that's because the CSS definition written here is already under `.reversehero` (check line: 219), so re-adding the classname `.reversehero` in the style definition doesn't match it with an appropriate element as the effective classname is `.reversehero .reversehero .button-alt` and `.reversehero .reversehero .mail-row p a` which doesn't exist.

Reply:
Okay, I got ii. I will recheck by today. Thanks again :)

Reply:
Hey @Nikhil-Ladha, thanks for guiding me. I have removed them. Can you please confirm if that's fine now?